crates/ruff_linter/src/registry/rule_set.rs

```diff
@@ -269,6 +270,22 @@ impl Debug for RuleSet {
     }
 }
 
+impl Display for RuleSet {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        if self.is_empty() {
+            write!(f, "[]")
+        } else {
+            write!(
+                f,
+                "[\n\t{}\n]",
+                self.iter()
+                    .map(|rule| format!("\"{rule:?}\""))
+                    .join(",\n\t")
+            )
+        }
+    }
```

Comment:
Would `f.debug_list` be useful here? 

Not that this part is performance sensitive but we can avoid allocating a string for each rule by using a for loop

```rust
write!(f, "[\n\t")?;

for rule in self.iter() {
	write!(f, "\"{rule:?}\"")?;
}

write!(f, "\n]")
```

Joking: We could use our formatter infrastructure to print the rules on a single line if they fit into an 80-character line length and split them when they do not :laughing:

Reply:
> Would `f.debug_list` be useful here?

Not really since it formats everything on one line 😞 

Thanks for the suggestion! I was using an allocating `format!` + `.join` to avoid a trailing comma, but apparently trailing commas are valid toml anyway, so my bad 🤷‍♀️